### PR TITLE
Comments: Reactions

### PR DESCRIPTION
**Pull request by marcbouchenoire**
This PR adds reactions to Comments, included in the default components and accessible via hooks when using primitives.

### `@liveblocks/react`

 - Add `useAddReaction`/`useCommentReaction` hooks.
 	- ~~API feedback: `addCommentReaction` vs `addReaction`?~~

### `@liveblocks/react-comments`

- Add reaction button to `Comment`
- Add reactions list within `Comment`
- Add emoji picker
- **Breaking (beta):** `Comment`’s `indentBody` and `Thread`’s
  `indentCommentBody` were renamed to `indentContent` and `indentCommentContent`
  respectively. `Thread`’s `onResolveChange` was renamed to `onResolvedChange`.

(I had an issue while rebasing to remove https://github.com/liveblocks/liveblocks/pull/1155/commits/32fecdae5764543a3f38feb7cf0ffc55fd426d2d and the branch's history is now completely broken, the contents of the commit are correctly reverted though and the example isn't linked anymore. Since we use squashed merge commits, the broken history won't affect `main`.)

### CODE REVIEW

**Review by CTNicholas — APPROVED** [review]
Docs look good 👍

File: packages/liveblocks-core/src/comments/index.ts [hunk 1]
```diff
@@ -42,6 +42,16 @@ export type CommentsApi<ThreadMetadata extends BaseMetadata> = {
     threadId: string;
     commentId: string;
   }): Promise<void>;
+  addCommentReaction(options: {
```
Comment (nvie) [inline packages/liveblocks-core/src/comments/index.ts:45]: Not sure if these names are set in stone already? Would we rename those to `addReaction` and `removeReaction` maybe? A tad bit shorter, yet just as clear and still unambiguous.

Comment (nvie) [reply packages/liveblocks-core/src/comments/index.ts:45]: (And from the params it's already clear they need to be attached to a comment.)

**Review by nvie — APPROVED** [review]
It looks amazing! 😍

File: packages/liveblocks-core/src/comments/index.ts (hunk 1 above)
Comment (marcbouchenoire) [reply packages/liveblocks-core/src/comments/index.ts:45]: Not in stone! I was wondering about this exactly: https://github.com/liveblocks/liveblocks/pull/1155#:~:text=API%20feedback%3A%20addCommentReaction%20vs%20addReaction%3F. I'll wait for more eyes but I'm leaning towards `addReaction`/`removeReaction` as well.

Comment (ofoucherot) [reply packages/liveblocks-core/src/comments/index.ts:45]: I also like `addReaction` and `removeReaction`!

**Review by ofoucherot — APPROVED** [review]
I agree with the proposed renaming for the methods `addReaction` and `removeReaction` but the PR is great 💯 👍 📦 🥇 🚀

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
+# v1.3.7
```
Comment (nvie) [inline CHANGELOG.md:1]:
```suggestion
# v1.4.0
```

Comment (marcbouchenoire) [reply CHANGELOG.md:1]: Yep I'm making another PR at the moment that fixes that and document the Yjs DevTools changes.

**Pull request merged**